examples/plugins/src/benchmark-js/src/benchmark.suite-runner.integration.test.ts

```diff
@@ -0,0 +1,23 @@
+import { describe, expect, it } from 'vitest';
+import { runner } from './benchmark.suite-runner';
+
+describe('benchmark runner', () => {
+  it('should execute valid suite', async () => {
+    await expect(
+      runner.run({
+        suiteName: 'suite-1',
+        targetImplementation: 'current-implementation',
+        cases: [
+          [
+            'current-implementation',
+            () => new Promise(resolve => setTimeout(resolve, 5)),
+          ],
+          [
+            'slower-implementation',
+            () => new Promise(resolve => setTimeout(resolve, 30)),
+          ],
+        ],
+      }),
+    ).resolves.toStrictEqual(expect.arrayContaining([]));
+  }, 20_000);
```

Comment:
Does this really need 20 seconds to run? :raised_eyebrow: 

Same goes for: `tinybench.suite-runner.integration.test.ts`

Reply:
down to `15_000`